nuxt.config.js

```diff
@@ -204,3 +214,34 @@ module.exports = {
     }
   }
 };
+
+if (Number(process.env['ENABLE_XX_USER_AUTH'])) {
+  config.auth = {
+    //redirect routes: 'callback' option for keycloak redirects,
```

Comment:
Be consistent with other comments' format, i.e. include a space after `//` and use sentence case.